### ISSUE TITLE
WatchConfig should also watch cert/pk

### ISSUE THREAD

**Issue by splitice** [glauth/glauth#234]
I'm investigating using lets encrypt to generate a valid CA signed certificate. A certificate like this expires every 90 days.

It would be ever so convenient if the `WatchConfig` option was expanded to include the certificate / pk file.

**Comment by Fusion** [comment]
Good point. This being said, the WatchConfig feature was created a while ago and I am not clear on how much testing was performed at the time. Do you have good experience with dynamic reload, outside the cert concern?

**Comment by splitice** [comment]
It works for me when adding users/groups.

**Comment by splitice** [comment]
One note I will add is that the watcher does not work recursively.

This can be worked around but its worth noting that k8s configmaps when mounted as volumes are nested symlinks (2 deep).
